content/400-pulse/200-getting-started.mdx

```diff
@@ -17,19 +16,39 @@ Prisma Pulse currently supports PostgreSQL. We'd love to hear [which databases](
 
 To get started with Pulse, you will need the following:
 
-- A [Prisma Data Platform workspace](https://console.prisma.io).
-- [Prisma Client](/orm/prisma-client) version `4.16.1` or higher and [`@prisma/extension-pulse`](https://www.npmjs.com/package/@prisma/extension-pulse) version `1.0.1` or higher.
-- A publicly accessible PostgreSQL (version 12+) database with [logical replication](https://www.postgresql.org/docs/current/logical-replication-quick-setup.html) enabled. View our [setup guide](/pulse/database-setup/general-database-instructions#enable-logical-replication) on configuring logical replication for your database.
+- An account in the [Prisma Data Platform](https://console.prisma.io).
+- The **connection string** of a Pulse-ready database (if you don't have one yet, you can configure your database the instructions [here](/pulse/database-setup) or [use a Railway template](https://railway.app/template/pulse-pg?referralCode=VQ09uv))
+
+## 1. Enable Pulse in the Platform Console
+
+### 1.1. Select workspace -> project -> environment
+
+Open the [Platform Console](https://console.prisma.io), navigate to your workspace of choice, then select the project and choose the environment in which you want to enable Pulse.
+
+### 1.2. Enable Pulse
+
+When you're there, click the **Enable Pulse** button.
 
-## 1. Enable Pulse
+### 1.3. Configure Pulse
 
-Navigate to your Prisma Data Platform project, choose an environment, and enable Pulse. We'll connect to your database and verify connectivity during setup.
+The Pulse **Setup** screen requires you to:
 
-> Once enabled, you'll be prompted to generate an API key that you'll use in your extended Prisma Client to authenticate requests. Store this API key in your application's `.env` file:
->
-> ```env file=.env showLineNumbers
-> PULSE_API_KEY="your_secure_pulse_api_key"
-> ```
+- provide your **Database connection string**
+- select a **Region** where Pulse should be hosted
+- decide whether you want to use the **Automatic setup** for **Database replication** (only available on paid plans)
+
+When you're done with that, click the **Enable Pulse** button at the bottom of the screen. This will test the connectivity to your database.
+
+
+### 1.4. Generate an API key
```

Comment:
Do you need to generate a new API key, even if you already have one for this environment?

Reply:
Not sure I follow exactly!

These are the "Getting started" instructions, which assume that people don't have an API key (for that particular environment) yet. If they had one, I believe they wouldn't be able to go through the previous steps where they set up Pulse (e.g. click the **Enable Pulse** button because once Pulse is enabled, this button says **Manage Pulse**).

When you set up Pulse for the first time using the instructions above, you do need to explicitly generate that API key. Just double-checked that workflow:

<img width="679" alt="image" src="https://github.com/prisma/docs/assets/4058327/5251fafb-ec01-44d2-8cc0-78daffd5b49a">

Reply:
So, you could skip this if you already have an API key. Should we mention that?

Reply:
Added this note:

<img width="545" alt="image" src="https://github.com/prisma/docs/assets/4058327/cbc95076-e30b-418e-8bc9-2c532d54b6de">